pnpm-lock.yaml

```diff
@@ -21446,13 +21455,16 @@ packages:
       fs-extra: 11.2.0
       glob: 7.2.3
       ignore: 5.3.0
+      json-schema-to-typescript: 15.0.1
```

Comment:
Hmm. I guess this should have been a devDependencies in build-tools, right?

Reply:
Yeah. I originally was thinking of using it at runtime and forgot to move the dep when I abandoned that idea. I'll PR
the change to build-tools.

Reply:
#22301